src/test/groovy/com/vanniktech/android/junit/jacoco/GenerationTest.groovy

```diff
@@ -138,12 +138,10 @@ public class GenerationTest {
 
             assert executionData.singleFile == project.file("${project.buildDir}/jacoco/test${flavor.capitalize()}${buildType.capitalize()}UnitTest.exec")
 
-            assert additionalSourceDirs.size() == 3
```

Comment:
can you assert against the new number and also add the missing contains so that we're not missing anything?